src/components/pages/UserPage.js

```diff
@@ -81,7 +78,7 @@ export const UserPage = ({ match }) => {
         ></input>
       </form>
 
-      <RepoList repos={repos} user={user} />
+      <RepoList repos={currentUserRepos} user={user} />
```

Comment:
mini nit can we remove the new line above. theres also a new line in JSX for RepoList file